donut-app/resources/layouts/layout.xml

```diff
@@ -1,5 +1,8 @@
 <layout id="MainLayout">
     <drawable class="Background" />
     <label id="TimeLabel" x="center" y="10" font="Gfx.FONT_LARGE" justification="Gfx.TEXT_JUSTIFY_CENTER" />
-    <label id="DistanceLabel" x="center" y="50" font="Gfx.FONT_LARGE" justification="Gfx.TEXT_JUSTIFY_CENTER" />
+    <drawable id="Donuts" class="MultipleDonutsDrawable">
+    	<param name="numBurned">0</param>
+    </drawable>
+    <label id="DistanceLabel" x="center" y="center" font="Gfx.FONT_LARGE" justification="Gfx.TEXT_JUSTIFY_CENTER" />
```

Comment:
I'm not sure if this is the best location `<center, center>` but I don't have a better default...